### DISCUSSION TITLE
Sunbird-Knowlg: Release-5.2.0

### DISCUSSION THREAD

**Discussion by Priyakashi** [Sunbird-Knowlg/Community#76] [General Discussions]
Release 5.2.0 is In-Progress and here are the details:

![image](https://user-images.githubusercontent.com/106071204/200742307-6cdbc618-8275-4e6e-8978-bc9ff7b08810.png)

![image](https://user-images.githubusercontent.com/106071204/200742580-3b70fa2b-27a4-4dea-8819-92c8aa5a9a15.png)

![image](https://user-images.githubusercontent.com/106071204/200742678-0f1773d5-6ac5-4a4b-aeb5-c4d2580bbb02.png)

![image](https://user-images.githubusercontent.com/106071204/200743076-c8522b64-37ec-4c8e-b99f-b01f7fdcbfe0.png)

@vinukumar-vs ,@pramodkvarma, @vrayulu, @SanthoshVasabhaktula, @rhwarrier,
@rahul-kankaria, @ChakshuGautam, @kiranharidas187, @pritamps, @chitranshu-keshav @rahul101001000 @coolbung
@surendrasinghs, @pallakartheekreddy, @krgauraw, @luckynvdu @reshmi-nair, @sharathkashyap, @gandham-santhosh, @kameshbhr, @AmiableAnil, @Jayaprakash8887, @anandp504,
@surendrasinghs
Shikshalokam: @vijiurs @aishwaryashikshalokam @Prateek-slokam @aks30 @varsha-sl @MeghnaSL
Samagra: @Prerna789 @aayushkedia95
SunbirdCB: @pkranga
Avanti: @pritamps @karn-nahata
DC's: @SanthoshVasabhaktula @vrayulu @maheshkumargangula
Sunbird QA: @ashokreddy1208
@gandham-santhosh
@kameshbhr @mmanimegalai
@madhucr @alok-os @anunagaraj @ShailandraOjha

**Comment by Naveenkumar739** [comment]
Hello All,

Please find the Sunbird knowledge Building block R5.2.0 sign-off here.

Environment: https://dev.knowlg.sunbird.org/

Testing Status: Completed

Defects Dashboard: [link](https://project-sunbird.atlassian.net/jira/software/c/projects/KN/issues/?jql=project%20%3D%20%22KN%22%20AND%20reporter%20IN%20%28currentUser%28%29%2C%2260f117ba43c956007055794e%22%29%20AND%20type%20IN%20%28%22Defect%22%2C%22Checklist-Item%22%29%20ORDER%20BY%20created%20DESC)

Execution Sheet: [link](https://docs.google.com/spreadsheets/d/16Zb9lZw9gLGdzugKF8gHlkoDW-vDMD0amY7afJ5wbT0/edit#gid=690730285)

Scope : [Link](https://project-sunbird.atlassian.net/issues/?filter=11287&jql=project%20%3D%20KN%20AND%20issuetype%20in%20standardIssueTypes()%20AND%20status%20in%20(Done%2C%20%22Failed%20Validation%22%2C%20%22In%20Development%22%2C%20%22In%20Validation%22%2C%20Open%2C%20%22To%20Do%22)%20AND%20labels%20in%20(QA_Not_Required%2C%20QA_Required%2C%20QA_Required_Regression)%20AND%20Sprint%20in%20(273%2C%20274)%20ORDER%20BY%20key%20ASC%2C%20created%20DESC)

BB Sign Off Template: [Link](https://docs.google.com/spreadsheets/d/1mu_liuEDb_x1JBABS6PYxQW5aPY5STNMJnb76eD7ejw/edit#gid=0)

cc @vinukumar-vs @anunagaraj @ShailandraOjha @chitranshu-keshav
